src/node_url.h

```diff
@@ -4,11 +4,18 @@
 #if defined(NODE_WANT_INTERNALS) && NODE_WANT_INTERNALS
 
 #include "node.h"
+#include "env.h"
+#include "env-inl.h"
+
 #include <string>
 
 namespace node {
 namespace url {
 
+using v8::MaybeLocal;
+using v8::Value;
+
+
 #define BIT_AT(a, i)                                                          \
```

Comment:
Aside from this PR, maybe we would want to move these private macros and inlined functions into the `.cc` file itself?